pages/dkp/konvoy/2.2/Install/index.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: layout.pug
+navigationTitle: Install
+title: Install
+menuWeight: 80
+excerpt: Install a Konvoy cluster
```

Comment:
Not sure, but I think this should be "Install Konvoy on a cluster" OR "Create a Konvoy cluster".